include/range/v3/view/all.hpp

```diff
@@ -103,8 +122,7 @@ namespace ranges
             RANGES_INLINE_VARIABLE(all_fn, all)
 
             template<typename Rng>
-            using all_t =
-                meta::_t<std::decay<decltype(all(std::declval<Rng>()))>>;
+            using all_t = decltype(all(std::declval<Rng>()));
```

Comment:
Note that this decay is unnecessary, all of `all_fn`'s `operator()` overloads return cv-and-ref-unqualified types.